apps/i18n/common/en_us.json

```diff
@@ -424,6 +424,8 @@
   "courseBlocksInternationalGradeBandsMiddleDescription": "As flexible courses that can be taught in a unit, semester, or full year, CS Discoveries and CS Fundamentals Express introduce students to app design, JavaScript programming, physical computing, and more.",
   "courseBlocksLegacyNotificationHeading": "CS Fundamentals Courses 1-4 & the Accelerated Course",
   "courseBlocksLegacyNotificationBody": "Were you using Courses 1-4 or the Accelerated Course? They’re still available! Our transition guide will help you find the right course for your classroom.",
+  "courseBlocksLegacyNotificationSupportEndedHeading": "Courses 1-4 & the Accelerated Course are going away!",
```

Comment:
Hey Turner, sorry I didn't update the spec to reflect this, but I think I'd like to change these lines to:
"Courses 1-4 & the Accelerated Course are outdated!"
and 
"Courses 1-4 and the Accelerated Course are still available, but we have better options now. Our transition guide will help you get started with Courses A-F, which teach the same concepts more effectively, are actively supported, and have full translation into over 30 languages."

Reply:
On it! [This PR](https://github.com/code-dot-org/code-dot-org/pull/45798) will update that text!